internal/auth/org/org.go

```diff
@@ -36,16 +36,18 @@ func Cmd() *cobra.Command {
 }
 
 func createCmd() *cobra.Command {
-	var title string
+	var title, namespace string
 	cmd := &cobra.Command{
 		Use:  "create",
 		Args: cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return create(cmd.Context(), title)
+			return create(cmd.Context(), title, namespace)
 		},
 	}
 	cmd.Flags().StringVar(&title, "title", "", "Title of the organization")
+	cmd.Flags().StringVar(&namespace, "kubernetes-namespace", "", "Kubernetes namesapce of the organization")
```

Comment:
just personal preference, but I'd like to give shortname
```suggestion
	cmd.Flags().StringVarP(&namespace, "kubernetes-namespace", "n", "", "Kubernetes namesapce of the organization")
```